fix_uv_lock.py

```
@@ -0,0 +1,28 @@
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)